templates/events/event-edit.html

```diff
@@ -521,6 +527,60 @@ <h5 type='text'>Location Address</h5>
 
           <!-- Script to handle changing header -->
           <script>
+              function groupdropdownmenushow(title, titlevalue, id, idvalue) {
```

Comment:
It would be better to use camelcase for function names for better readability. Thanks.

Reply:
fixed